### ISSUE TITLE
Fix a11y issue with footer missing in no-js pages

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#8716]
![Image](https://user-images.githubusercontent.com/110461050/196916238-f43bf9f8-b604-4069-8850-46949a1001dc.png)

**Comment by rcantin-w** [comment]
This needs a longer look than we thought, so [a new ticket was created in the access board](https://github.com/wellcomecollection/wellcomecollection.org/issues/8717) to implement a solid fix:
